hazelcast-jet-distribution/src/main/resources/bin/common.sh

```diff
@@ -38,6 +38,14 @@ if [ "x$JAVA_OPTS_DEFAULT" != "x" ]; then
     JAVA_OPTS="${JAVA_OPTS_DEFAULT} ${JAVA_OPTS}"
 fi
 
+if [ "x$JET_LICENSE_KEY" != "x" ]; then
+	JAVA_OPTS="$JAVA_OPTS -Dhazelcast.enterprise.license.key=${JET_LICENSE_KEY}"
+fi
+
+if [ "x$JET_LICENCE_KEY" != "x" ]; then
```

Comment:
the if is duplicated, also the indentantion on the various options should be aligned. I would also group the statements effecting JAVA_OPTS and CLASSPATH so it looks bit more organized

Reply:
It is not duplicated but covering the spelling options of `licen(c|s)e`

Reply:
why do we need two different properties though?

Reply:
The outcome of this https://github.com/hazelcast/hazelcast-docker/issues/68 was to accept both spellings for user convenience.

Reply:
ok, at least the indentation of the file should be fixed / cleaned up. I'm also not sure why we use `"x$JET_LICENSE_KEY" != "x"` rather than `if [ -n "$VAR" ]`

Reply:
I agree with the indentation, I'll fix it. For the other concern see https://stackoverflow.com/a/174288 for more detailed answer.